myapp/src/Day2/Que16/EmployeeSalary.jsx

```diff
@@ -0,0 +1,52 @@
+// 16.Create a functional component named EmployeeSalary that displays the average salary of a list of employees.
+// Define an array of employee objects, where each object has a name and salary property.
+// Use the useMemo hook to calculate the average salary of employees.
+// Ensure that the useMemo hook has a dependency on the employee data so that it recalculates when the employee data changes.
+// Render the average salary on the screen.
+// Include a button that, when clicked, updates the employee data with new salaries.
```

Comment:
@anantmishra-successive this is not done. plz don't leave things pending. As I have told earlier, you should test before raising PR.